oci/spec_opts_test.go

```diff
@@ -780,3 +780,83 @@ func TestWithWindowsDevice(t *testing.T) {
 		})
 	}
 }
+
+func TestWithLinuxDeviceSymlinked(t *testing.T) {
```

Comment:
This test needs to be an an `_linux_test.go` file (windows is failing; see below)